backend/services/applicationService.py

```diff
@@ -44,6 +44,20 @@ def registerApplication(comments, needs, user, partnerUsername, seatRollover, pr
         return "", 400
 
 
+def updateApplication(userid, form):
+    try:
+        application = getApplicationByUserId(userid)
+        for field in form.keys():
+            if(form[field]):
```

Comment:
Hm, so replace it with `if field in application`?

Reply:
idk. We probably should do a better check. Since there is some fields of the application the user should not be able to edit. For example their study-rank, which i'm currently implementing

Reply:
What do you think about creating a method on Application that returns user-editable fields?
Something like:
```python
class Application:
...
  def userEditableFields(self):
    return ["comments", "needs", "seatRollover", ...]
```

Reply:
Smart :lightbulb: